pkg/scmprovider/pull_requests.go

```diff
@@ -35,15 +35,15 @@ func (c *Client) populateFields(ctx context.Context, pr *scm.PullRequest, owner,
 		}
 		pr.Labels = append(pr.Labels, labels...)
 	}
-	// If we have an ID but no name, we've probably got a GitLab PR so go get its base information
-	if pr != nil && pr.Base.Repo.ID != "" && pr.Base.Repo.Name == "" {
+	// If we have an ID but no name or no clone URI, we've probably got a GitLab PR so go get its base information
+	if pr != nil && pr.Base.Repo.ID != "" && (pr.Base.Repo.Name == "" || pr.Base.Repo.Clone == "") {
 		baseRepo, _, err := c.client.Repositories.Find(ctx, pr.Base.Repo.ID)
 		if err != nil {
 			return nil, errors.Wrapf(err, "getting base repository for PR")
 		}
 		pr.Base.Repo = *baseRepo
 	}
-	if pr != nil && pr.Head.Repo.ID != "" && pr.Head.Repo.Name == "" {
+	if pr != nil && pr.Head.Repo.ID != "" && (pr.Head.Repo.Name == "" || pr.Head.Repo.Clone == "") {
```

Comment:
if the above is GitLab what case is this then?

Reply:
The above is for the base repo, this one is for the head repo - they're both gitlab cases. But I'm gonna go fix that up in go-scm now and put this on hold for the moment anyway.

Reply:
fix is at https://github.com/jenkins-x/go-scm/pull/199 - doing a verification here now.